tests/StatsdClient.Tests/DogStatsdServiceTests.cs

```diff
@@ -141,7 +140,7 @@ public void setting_prefix_starttimer()
 
                 var metricTimeInMs = Convert.ToInt32(metricTimeInMsSplit[0]);
                 Assert.IsTrue((metricTimeInMs >= 1000), "Processing should have taken at least 1000ms");
-                Assert.IsTrue((metricTimeInMs < 1100), "Timer reported 10% higher than time taken in action");
+                Assert.IsTrue((metricTimeInMs < 1300), $"Timer reported 30% higher than time taken in action: {metricTimeInMs} VS 1300");
```

Comment:
Do you suggest removing this test?

Reply:
Yes, I'm wondering if you think this test is indeed useful or not necessarily. I think that the fact that it could be affected by load, underlying hardware, etc... makes it less useful than perhaps intended.

Reply:
This test checks that we send the duration of the action for `StartTimer`.
What do you think about keeping it for now and remove it if it fails again?